tests/test_shopcarts.py

```diff
@@ -1,32 +1,46 @@
 """
```

Comment:
as this file is for testing the code in models.py, I think the naming should align, e.g. test_models.py

Reply:
I'm not sure about this because Professor mentioned in the video that if there are multiple classes he would go create multiple tests, one for each class and he would not put them all in one big model. I guess we can ask Professor for this.

Reply:
good idea! let's consult Professor about this 🙂

Reply:
According to Professor's [response](https://nyu-devops-summer23.slack.com/archives/C05BNCWBJBW/p1687968290419039?thread_ts=1687923382.484859&cid=C05BNCWBJBW) and our scenario (where all models are in `models.py`), we're recommended to put all the test cases for `models.py` in `test_models.py`

Reply:
Agree, I will gather the two test files into test_models.py.